markdown/2017-10-23.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Weekly Digest
+show_social: true
+show_agenda: true
+show_sponsorship: true
+...
+
+---
+
+<center>
+<div style="position:relative;width:50%">
+    ![Want want biscuit](https://i.imgur.com/VTZIP2c.png "I didn't know this biscuit existed until today. Interesting combination of salty and sweet, with a rice-y aftertaste. 8/10 would eat again.")
+</div>
```

Comment:
Unattributed quote, plagiarism?

Reply:
It's a string, not a quote. If I was to attribute anyone it's myself